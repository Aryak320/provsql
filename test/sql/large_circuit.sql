\set ECHO none
\pset format unaligned

CREATE table t(x INT);
INSERT INTO t VALUES(generate_series(0,9999));
SELECT provsql.add_provenance('t');
DO $$ BEGIN
    PERFORM provsql.set_prob(provsql.provenance(),0.5) from t;
END $$;
CREATE TABLE tresult AS
SELECT provsql.probability_evaluate(provsql.provenance()) from (select distinct 1 from t) temp;
SELECT provsql.remove_provenance('tresult');
SELECT * FROM tresult;
DROP table t;
DROP table tresult;
