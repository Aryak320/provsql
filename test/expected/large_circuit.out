\set ECHO none
add_provenance

(1 row)
remove_provenance

(1 row)
probability_evaluate
1
(1 row)
